src/Conversions/Casing/PythonCaseConverter.ts

```diff
@@ -0,0 +1,40 @@
+import { PackageLowerCaseConverter } from "./PackageLowerCaseConverter";
+
+/**
+ * Converts a series of words to PythonCase.
```

Comment:
In the other case converter files (unless I missed one), these comments use the same case as the converter. Something like `.python/case`/`.python/import/case` (or whatever a more proper example would be).